test/Sia.Gateway.Tests/TestDoubles/FakeGatewayException.cs

```diff
@@ -2,7 +2,9 @@
 
 namespace Sia.Gateway.Tests.Middleware
 {
+#pragma warning disable CA1032 // Implement standard exception constructors
```

Comment:
I turned this off in the global suppressions because any custom exceptions will likely follow this rule so we don't need to wrap all exception classes in comments: https://github.com/Azure/Sia-Root/blob/master/Domain/Sia.Core/GlobalSuppressions.cs#L20